tcav/tcav.py

```diff
@@ -186,7 +186,7 @@ def __init__(self,
     self.params = self.get_params()
     tf.logging.info('TCAV will %s params' % len(self.params))
 
-  def run(self, num_workers=10, run_parallel=False):
+  def run(self, num_workers=10, run_parallel=False, return_proto=False):
```

Comment:
update docstring here for new param and possible return val

Reply:
done